src/screencast/pipewire_screencast.c

```diff
@@ -229,7 +229,27 @@ int xdpw_pwr_core_connect(struct xdpw_state *state) {
 	return 0;
 }
 
+void xdpw_pwr_context_destroy(struct xdpw_state *state) {
+	struct xdpw_screencast_context *ctx = &state->screencast;
+
+	logprint(DEBUG, "pipewire: disconnecting fom core");
+
+	if (ctx->core) {
+		pw_core_disconnect(ctx->core);
+		ctx->core = NULL;
+	}
+
+	if (ctx->pwr_context) {
+		pw_context_destroy(ctx->pwr_context);
+		ctx->pwr_context = NULL;
+	}
+}
+
 void xdpw_pwr_stream_destroy(struct xdpw_screencast_instance *cast) {
```

Comment:
Nit: Could you please move this function up, so we have the same order as in the header?